<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QGroupBox::title {
        subcontrol-origin: margin;
        subcontrol-position: top center; /* other options: top left, top right */
        padding: 0 10px;
        color: #2980b9;
}
QDateEdit {
    background-color:rgba(255,255,255,0.5);
    border: 1px solid #444;
    border-top-color: #666;
    border-left-color: #666;
    border-bottom-color: #222;
    border-right-color: #222;
    border-radius: 4px;
    color: #f1f5f9;
    padding: 6px 10px;
    font-size: 16px;
}

QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: right center;
    width: 25px;
    background:#2980b9;
}

QDateEdit::down-arrow {
    image: none;
    font-size: 14px;
    color: white;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color : white;
font-weight : bold;
</string>
       </property>
       <property name="title">
        <string>from</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QDateEdit" name="dateEditFrom">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">height:50px;
</string>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="time">
           <time>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
           </time>
          </property>
          <property name="displayFormat">
           <string>dd/MM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../assets/calendar - Copy.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true">color : white;
font-weight : bold;</string>
       </property>
       <property name="title">
        <string>to</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QDateEdit" name="dateEditTo">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">height:50px;
</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="time">
           <time>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
           </time>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::YearSection</enum>
          </property>
          <property name="displayFormat">
           <string>dd/MM/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="currentSectionIndex">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
        font-family: 'Arial';
        font-size: 14px;
    }</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>days present</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>days absent</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>total time (h)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>avg time (h)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>exit rate</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="export_csv_btn">
     <property name="styleSheet">
      <string notr="true">width:200px;
height:45px;
text-align:center;
background:#2980b9;
font-size:18px;
font-weight:bold;

</string>
     </property>
     <property name="text">
      <string>Export csv</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
